src/trove_classifiers/__init__.py

```diff
@@ -748,3 +750,7 @@
     "Natural Language :: Ukranian": ["Natural Language :: Ukrainian"],
     "Topic :: Communications :: Chat :: AOL Instant Messenger": [],
 }
+
+
+def classifiers_sorted(classifiers):
+    return natsorted(classifiers)
```

Comment:
It would be nice if this library had a static, pre-sorted list of classifiers (maybe called `sorted_classifiers`) to avoid the need to sort this same list over and over whenever it's needed. At one point this library had the ability to generate this file from some other canonical list, but it ended up being unnecessary. Maybe it's worth bringing back for this?

Reply:
I agree -- it's probably better as a linter job, than as a runtime task. That said, it's all mostly aesthetics. :)

Reply:
The CI already lints to check the `classifiers` set is in order:

`make lint` -> `python bin/sort.py src/trove_classifiers/__init__.py`


I've added a static, pre-sorted list in 9f84827ca341b7b2874edbbc6aa6ac694d04c3ef.